lib/api/graph.js

```diff
@@ -108,13 +108,69 @@ function createEmbed(graphJSON, options, callback){
     client.request("POST", "/graph/embed", params, callback);
 }
 
+/*section: graph
+ *comment: delete an embed with a specific id
+ *params:
+ *  embedId: the id of the embed to delete
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  var embedid = "foo";
+ *  dogapi.graph.deleteEmbed(embedid, function(err, res){
+ *    console.dir(res);
+ *  });
+ *  ```
+ */
+function revokeEmbed(embedId, callback){
+    client.request("GET", "/graph/embed/" + embedId + "/revoke", callback);
+}
+
+
+/*section: graph
+ *comment: get all embeds from datadog
+ *params:
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  dogapi.graph.getEmbeds(function(err, res){
+ *    console.dir(res);
+ *  });
+ *  ```
+ */
+function getEmbeds(callback) {
```

Comment:
hmm... actually... this might need `Embed` in it, since we aren't getting a `graph`.

we probably should have just created `embed.js` rather than putting these under `graph`, but that is probably fine for now...

ignore the previous comment `getEmbeds` is fine.